### ISSUE TITLE
More color folder options

### ISSUE THREAD

**Issue by bungadrum** [vinceliuice/Qogir-icon-theme#135]
Can you add more color options similar to what Tela has? The default blue is OK, but over time you kinda get tired of it

**Comment by Kingproone** [comment]
On that note, could something like dynamic coloring be doable? Kde's breeze theme does that, don't know which would be easier to maintain.

**Comment by soyteen** [comment]
Yes, this would be really useful and I hope it gets added. I'm using this icon theme as recommended by Matcha-gtk-theme, which supports different accent colors, including red and I would like for the icons to match.
